mobile/src/main/java/org/hawkular/client/android/adapter/TriggersAdapter.java

```diff
@@ -114,6 +119,9 @@ public void onClick(View view) {
         @BindView(R.id.text_wrapper)
         LinearLayout linearLayout;
 
+        @BindView(R.id.triggerBox)
```

Comment:
Remove `LinearLayout dataBox` and corresponding `setOnClickListener`

Not needed